networking/network_policy/about-network-policy.adoc

```diff
@@ -16,12 +21,15 @@ include::modules/nw-networkpolicy-optimize.adoc[leveloffset=+1]
 == Next steps
 
 * xref:../../networking/network_policy/creating-network-policy.adoc#creating-network-policy[Creating a network policy]
+ifdef::openshift-enterprise[]
```

Comment:
You could consider using `ifndef::openshift-dedicated,openshift-rosa[]` instead, in case the enclosed content is required for other distros, such as OKD.